modules/UI/videolayout/ConnectionIndicator.js

```diff
@@ -198,6 +198,9 @@ ConnectionIndicator.prototype.generateText = function () {
                 }
             }
 
+            // All of the transports should be either P2P or JVB
+            const isP2P = this.transport.length ? this.transport[0].p2p : false;
```

Comment:
Currently we can have either P2P or JVB transport advertised through the stats. Even if I would try to mark p2p on per transport entry it's impossible with the current UI and it's implementation (see code above).